runtime/doc/windows.txt

```diff
@@ -619,7 +619,7 @@ The minimal height and width of a window is set with 'winminheight' and
 41. :buffers		list of buffers
 
 The meaning of [N] depends on the command:
- [N] is number of buffers to go forward/backward on ?2, ?3, and ?4
```

Comment:
"?" is a wildcard. So, this means 2, 12, 22, 32, 3, 13, 23, 33, 4, 14, 24 and 34.

Reply:
oh, that confused me as well. It should be mentioned that ? is used as wildcard there.